### ISSUE TITLE
due to: Not able to find suitable driver for datasource

### ISSUE THREAD

**Issue by nickscut** [ClickHouse/clickhouse-jdbc-bridge#142]
![image](https://user-images.githubusercontent.com/5253435/157605835-5e8ced41-84ab-41f6-a151-cb212647c296.png)

select * from jdbc('jdbc:mysql://rm-xxx.mysql.rds.aliyuncs.com:3306/davinci?useUnicode=true&user=xx&password=xxx','xxx','select * from cron_job');

Such SQL will report an error

select * from jdbc('mysql5', 'xxx', 'select * from cron_job')

Such SQL runs fine

Our mysql is dynamic, there is no way to configure it in datasources in advance

**Comment by nickscut** [comment]
select * from jdbc('jdbc:mysql://rm-xxx.mysql.rds.aliyuncs.com:3306/xxx?useUnicode=true&user=xx&password=xxx','xxx','select * from cron_job');

Why does this SQL run fail ？

**Comment by nickscut** [comment]
![image](https://user-images.githubusercontent.com/5253435/157607701-ce52d3e3-e768-4c05-8ae7-75fe3051230c.png)
I follow this documentation using

**Comment by zhicwu** [comment]
> select * from jdbc('jdbc:mysql://rm-xxx.mysql.rds.aliyuncs.com:3306/xxx?useUnicode=true&user=xx&password=xxx','xxx','select * from cron_job');
> 
> Why does this SQL run fail ？

Have you tried `user:password@host_or_host_sublist` as mentioned at [here](https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-jdbc-url-format.html#connector-j-url-user-credentials)?

**Comment by nickscut** [comment]
select * from jdbc('mysql://davinci:Davinci123@rm-xx.mysql.rds.aliyuncs.com:3306/davinci','davinci','select * from cron_job');

I make sure user name and password are URL encoded.

![image](https://user-images.githubusercontent.com/5253435/157611329-8b301915-bfe7-44e2-9610-66585728ca80.png)

**Comment by zhicwu** [comment]
Did you forgot `jdbc:` prefix? It should `jdbc:mysql:...` not `mysql:...`?

**Comment by nickscut** [comment]
select * from jdbc('jdbc:mysql://davinci:Davinci123@rm-xxx.mysql.rds.aliyuncs.com:3306/davinci?useUnicode=true','davinci','select * from cron_job');

![image](https://user-images.githubusercontent.com/5253435/157613137-195de2df-b130-4faf-8bbb-120c4d6ce5a4.png)

**Comment by zhicwu** [comment]
The URL seems correct. Where did you put the MySQL JDBC driver?

**Comment by nickscut** [comment]
![image](https://user-images.githubusercontent.com/5253435/157613576-33cc3a2d-a72f-40c9-9983-781e3c8a8c40.png)
Do I have to define such a json file?

**Comment by nickscut** [comment]
We have a lot of mysql, and they are added dynamically, there is no way to define the json file in advance

**Comment by nickscut** [comment]
If the json file is not defined, how to put the driver in the bridge?

**Comment by nickscut** [comment]
select * from jdbc('mysql5', 'davinci', 'select * from cron_job')

This SQL statement works fine，mysql5 is configured in advance in the mysql5.json file

but：We have a lot of mysql, and they are added dynamically, there is no way to define the json file in advance

**Comment by zhicwu** [comment]
> If the json file is not defined, how to put the driver in the bridge?

You can put the driver in `lib` or any other directory and make sure it's included in CLASSPATH.

**Comment by nickscut** [comment]
Does it support K8S deployment?

Is there a complete chart file？

**Comment by nickscut** [comment]
@zhicwu  Does bridge support k8s deployment? Is there a complete chart file？

**Comment by nickscut** [comment]
![image](https://user-images.githubusercontent.com/5253435/158126762-0e4b1b11-1d47-4c12-bcd4-13f0cbc874ee.png)

It seems that there is no official k8s chart file for the time being

**Comment by zhicwu** [comment]
Yes, but it should be able to come up with one quickly based on docker-compose file - please feel free to submit a pull request.
